### PR TITLE
SK-401 update readme to include content on update data through collect element in js sdk

### PR DESCRIPTION
**Pull request by skyflow-bharti**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -599,6 +600,212 @@ cvvElement.mount('#cvv'); //Assumes there is a div with id='#cvv' in the webpage
...
+
+```javascript
+const collectElement = {
+ table: "string", // Required, the table this data belongs to.
+ column: "string", // Required, the column into which this data should be updated.
+ type: Skyflow.ElementType, // Skyflow.ElementType enum.
+ inputStyles: {}, // Optional, styles that should be applied to the form element.
+ labelStyles: {}, // Optional, styles that will be applied to the label of the collect element.
+ errorTextStyles: {}, // Optional, styles that will be applied to the errorText of the collect element.
+ label: "string", // Optional, label for the form element.
```
Comment (Shaik-Luqmaan) [inline README.md:625]: All the comments should start from the same point. Check other comments in the README

File: README.md [hunk 2]
```diff
@@ -599,6 +600,212 @@ cvvElement.mount('#cvv'); //Assumes there is a div with id='#cvv' in the webpage
...
+const options = {
+ required: false, // Optional, indicates whether the field is marked as required. Defaults to 'false'.
+ enableCardIcon: true, // Optional, indicates whether  the element needs a card icon (only applicable for CARD_NUMBER ElementType).
+ format: String, // Optional, format for the element (only applicable currently for EXPIRATION_DATE ElementType).
+ enableCopy: false, // Optional, enables the copy icon in collect and reveal elements to copy text to clipboard. Defaults to 'false').
+};
+const element = container.create(collectElement, options);
+```
+The `table` and `column1` fields indicate which table and column the Element corresponds to.
+
```
Comment (Shaik-Luqmaan) [inline README.md:640]: Pls give valid column names, not as column1

File: README.md [hunk 3]
```diff
@@ -599,6 +600,212 @@ cvvElement.mount('#cvv'); //Assumes there is a div with id='#cvv' in the webpage
...
+     table: "string", // Table name
+     column: "value", // Unique column in the table
+   },
+ ], // Optional
+};
+container.collect(options);
+```
+**Note:** `skyflowID` is required if you want to update the data. If `skyflowID` isn't specified, the `collect(options?)` method creates a new record in the vault.
+
+### End to end example of collecting data with Skyflow Elements
```
Comment (Shaik-Luqmaan) [inline README.md:703]: The title should be "Complete example of updating through Skyflow Elements"

**Review by Shaik-Luqmaan — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (skyflow-bharti) [reply README.md]: changes done

File: README.md (hunk 2 above)
Comment (skyflow-bharti) [reply README.md]: changes done

File: README.md (hunk 3 above)
Comment (skyflow-bharti) [reply README.md]: changes done

**Review by Shaik-Luqmaan — APPROVED** [review]

**Pull request merged**